### DISCUSSION TITLE
Welcome to Farcaster Protocol Discussions!

### DISCUSSION THREAD

**Discussion by varunsrin** [farcasterxyz/protocol#6] [General]
## 👋 Welcome!

We’re using Discussions as a place to: 

  * Ask questions about the protocol.
  * Share ideas and improvements. 
  * Report bugs or other issues.

**Comment by YashKarthik** [comment]
hey @varunsrin I'd like to understand how fc stores data:

1. How does fc associate likes and recasts to the person who liked / recasted. The cast meta object contains number of recasts, but doesn't mention who.
2. How does fc store connections like follows, watching.

Cheers!
Yash

**Reply by varunsrin** [reply]
for v1 objects (which are in the current app), recasts and likes are client level features and not protocol level features.

in v2 they will be protocol level features as described here: https://github.com/farcasterxyz/protocol#4-replication

**Reply by YashKarthik** [reply]
got it :)

thanks for the quick reply!

**Comment by YashKarthik** [comment]
Another quick question:

- How does the fc client recognize cast urls. Is it `farcaster://casts/merkleroot/merkleroot`

**Reply by YashKarthik** [reply]
cc @varunsrin

**Comment by davids078** [comment]
hello my acc is shadow banned https://warpcast.com/david07  my posts are not visible to public please fix

@varunsrin

**Comment by MN8611** [comment]
how to stop farcaster node from vps

**Comment by JohnSon2804** [comment]
Hey, good day 
I'm trying to authenticate my x with farcaster but is doesn't work, it says I'm not eligible to do that .

What requirements do I need to meet to be able to connect my x with farcaster .

**Comment by abiemp2-web** [comment]
I can't access my Farcaster account after the account recovery. I was told to wait 72 hours, and it's been more than 72 hours now, but I still can't log in.

**Comment by Waleedsh054** [comment]
Hi Farcaster Support Team,
I am facing a persistent issue with my account recovery. Here are the details:
The Issue: I initiated the account recovery process using my recovery phrase.
The Wait Period: I waited for the mandatory 72-hour security period to complete.
Current Status: Even after the 72 hours have passed, the recovery phrase is still not working. I am unable to regain access to my account, and the app is not proceeding to the login stage.
Steps Taken: I have verified that the phrase is entered correctly (all lowercase, correct order) and my internet connection is stable.
Could you please check if there is an on-chain synchronization issue or a backend glitch preventing my recovery from finalizing?

Account Details:
Farcaster Username: @waleedsh
Device: [ pixe 8 pro Android 16]
App Version: 2.0.18
I look forward to your guidance on how to resolve this.

Regards,
Waleed
